src/api/items.test.tsx

```diff
@@ -298,4 +300,107 @@ describe('catalogue items api functions', () => {
       );
     });
   });
+
```

Comment:
Just a note the describe at the top of the file should be `items api functions`.